packit_service/worker/build/copr_build.py

```diff
@@ -177,6 +178,13 @@ def run_copr_build(self) -> HandlerResults:
         self.copr_build_model.build_id = build_metadata.copr_build_id
         self.copr_build_model.save()
 
+        # release the hounds!
+        celery_app.send_task(
+            "task.babysit_copr_build",
+            args=(build_metadata.copr_build_id,),
+            countdown=60,
+        )
```

Comment:
So we have one-baby-sit task for all chroots?

Reply:
yes, that's how copr werks